pyfftw/interfaces/scipy_fftpack.py

```diff
@@ -580,12 +580,10 @@ def _dst(x, type=2, n=None, axis=-1, norm=None, overwrite_x=False,
                          '"ortho" or "forward".')
 
     if type == 3 and norm == 'ortho':
-        x = numpy.copy(x)
+        x = numpy.copy(x) / math.sqrt(2*x.shape[axis])
```

Comment:
Does this result in an additional copy? Should we do it explicitly in-place (`x /= ...`)?

Reply:
I changed this to use `/=`!